manifest.json

```
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "open-graph",
-  "version": "1.2.1",
+  "version": "1.2.2",
```

This is not necessary, because it is updated automatically.
```suggestion
  "version": "1.2.1",
```